### PR TITLE
Ignore k8s related updates in renovate

### PR DESCRIPTION
**Pull request by h3poteto**
Because I will update it manually.

### CODE REVIEW

**Review by sc-yan — APPROVED** [review]

File: renovate.json [hunk 1]
```diff
@@ -2,5 +2,12 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "ignoreDeps": [
+        "k8s.io/*"
```
Comment (sc-yan) [inline renovate.json:9]: Error type: The renovate configuration file contains some invalid settings
Message: packageRules[0]: Each packageRule must contain at least one match* or exclude* selector. Rule: {"ignoreDeps":["k8s.io/*"]}

@h3poteto  looks like ignoreDeps should be outside of packageRules

Comment (h3poteto) [reply renovate.json:9]: wow

Comment (h3poteto) [reply renovate.json:9]: how did find that error message?

Comment (sc-yan) [reply renovate.json:9]: https://github.com/oviceinc/rollout-optimizer-controller/issues/15#event-15415499948

Comment (h3poteto) [reply renovate.json:9]: oh

**Pull request merged**